### ISSUE TITLE
Minor regression: float settings display as percents

### ISSUE THREAD

**Issue by ericlee4** [Rune580/RiskOfOptions#40]
After #37 existing float settings are displaying as percents and I'm unable to view increment changes of <1 (though the underlying value does update correctly). Is there a way to opt out of this behavior or can it be limited to settings with coarser increments?

Minimal example
```cs
public class Plugin : BaseUnityPlugin {
  private void Awake() {
     var focalLength = Config.Bind("Post-Processing", "Focal Length", 0.1f, "The focal length");
     ModSettingsManager.AddOption(new StepSliderOption(focalLength,
        new StepSliderConfig {
           min = 0, max = 500, increment = 0.01f
        }));
  }
}
```
![image](https://github.com/user-attachments/assets/c69db2f9-68ff-4439-9828-6140f84541ab)

Here's an example of this occurring in practice where most of the settings use an increment of 0.01f
https://github.com/RiskOfResources/photomode/blob/master/PhotoMode/PhotoModeSettings.cs
![image](https://github.com/user-attachments/assets/affbb45f-1bb0-4361-972e-ba58dd690c62)

**Comment by Rune580** [comment]
Not sure how I managed to break that, I'll fix it as soon as I can.

**Comment by Rune580** [comment]
fixed by 2.8.2
